src/blackd/__init__.py

```diff
@@ -174,10 +174,8 @@ def parse_python_variant_header(value: str) -> Tuple[bool, Set[black.TargetVersi
                     raise InvalidVariantHeader("major version must be 2 or 3")
                 if len(rest) > 0:
                     minor = int(rest[0])
-                    if major == 2 and minor != 7:
-                        raise InvalidVariantHeader(
-                            "minor version must be 7 for Python 2"
-                        )
+                    if major == 2:
+                        raise InvalidVariantHeader("Python 2 is unsupported")
```

Comment:
```suggestion
                        raise InvalidVariantHeader("Python 2 is not supported")
```
NIT: my bad if it really means something like "support was removed", but this sounds off to me.